### PR TITLE
chore(RHTAPREL-667): remove all CR default values from tasks

### PR DESCRIPTION
**Pull request by theflockers**
This commit removes default CR snapshotPath and dataPath
values from tasks.

Signed-off-by: Leandro Mendes <lmendes@redhat.com>

### CODE REVIEW

**Review by johnbieren — COMMENTED** [review]
Changes look good. In the commit body, RPA path defaults are removed too, not just data and snapshot. Also, I think the task READMEs need update as they typically have the default value stored

**Comment by theflockers** [comment]
> Changes look good. In the commit body, RPA path defaults are removed too, not just data and snapshot. Also, I think the task READMEs need update as they typically have the default value stored

Sure thing. I moved it to ready because I wanted e2e to run, but I will do the requested changes. Thanks for saying!

**Comment by johnbieren** [comment]
> > Changes look good. In the commit body, RPA path defaults are removed too, not just data and snapshot. Also, I think the task READMEs need update as they typically have the default value stored
> 
> Sure thing. I moved it to ready because I wanted e2e to run, but I will do the requested changes. Thanks for saying!

ack, fyi you can do `/test all` on a draft pr to run the prow tests without moving it to ready

File: tasks/check-data-keys/README.md [hunk 1]
```diff
@@ -12,9 +12,12 @@ Currently, `releaseNotes` is the only supported system.
 
 | Name     | Description                                             | Optional | Default value |
 |----------|---------------------------------------------------------|----------|---------------|
-| dataPath | Path to the JSON string of the merged data to use       | Yes      | data.json     |
+| dataPath | Path to the JSON string of the merged data to use       | No       |               |
 | systems  | The systems to check that all data keys are present for | Yes      | []            |
 
+## Changes in 0.5.1
+- Remove default value for `datPath`
```
Comment (johnbieren) [inline tasks/check-data-keys/README.md]: typo - datPath -> dataPath
Also, the other tasks you bumped the y of the version but here the z

Comment (theflockers) [reply tasks/check-data-keys/README.md]: fixed!

File: tasks/embargo-check/README.md
Comment (johnbieren) [inline tasks/embargo-check/README.md:1]: Missing changelog entry

File: tasks/prepare-fbc-release/README.md [hunk 2]
```diff
@@ -11,5 +11,8 @@ to each component, so other task can use them.
 
 | Name         | Description                                                             | Optional | Default value      |
 |--------------|-------------------------------------------------------------------------|----------|--------------------|
-| snapshotPath | Path to the JSON string of the Snapshot spec in the data workspace      | Yes      | snapshot_spec.json |
-| dataPath     | Path to the JSON string of the merged data to use in the data workspace | Yes      | data.json          |
\ No newline at end of file
+| snapshotPath | Path to the JSON string of the Snapshot spec in the data workspace      | No |  |
```
Comment (johnbieren) [inline tasks/prepare-fbc-release/README.md]: Can you fix the spacing so the `|` still aligns with row 13?

File: tasks/update-ocp-tag/README.md
Comment (johnbieren) [inline tasks/update-ocp-tag/README.md:1]: Missing changelog entry

File: tasks/validate-single-component/validate-single-component.yaml
Comment (johnbieren) [inline tasks/validate-single-component/validate-single-component.yaml:1]: Missing version bump

File: tasks/create-advisory/create-advisory.yaml
Comment (johnbieren) [inline tasks/create-advisory/create-advisory.yaml:1]: Missing README and version update

File: tasks/create-internal-request/create-internal-request.yaml
Comment (johnbieren) [inline tasks/create-internal-request/create-internal-request.yaml:1]: Missing README and version update

File: tasks/create-pyxis-image/create-pyxis-image.yaml
Comment (johnbieren) [inline tasks/create-pyxis-image/create-pyxis-image.yaml:1]: Missing README and version update

File: tasks/run-file-updates/run-file-updates.yaml
Comment (johnbieren) [inline tasks/run-file-updates/run-file-updates.yaml:1]: Missing README and version update

File: tasks/sign-index-image/sign-index-image.yaml
Comment (johnbieren) [inline tasks/sign-index-image/sign-index-image.yaml:1]: Missing README and version update

File: tasks/slack-webhook-notification/slack-webhook-notification.yaml
Comment (johnbieren) [inline tasks/slack-webhook-notification/slack-webhook-notification.yaml:1]: Missing README and version update

**Comment by theflockers** [comment]
Thanks for the detailed review, @johnbieren. Impressive eyes. PTAL!

File: tasks/create-internal-request/README.md
Comment (johnbieren) [inline tasks/create-internal-request/README.md:1]: dataPath default value still present on line 12

File: tasks/create-pyxis-image/create-pyxis-image.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ kind: Task
 metadata:
   name: create-pyxis-image
   labels:
-    app.kubernetes.io/version: "2.2.2"
+    app.kubernetes.io/version: "2.3.2"
```
Comment (johnbieren) [inline tasks/create-pyxis-image/create-pyxis-image.yaml]: This should be 2.3.0

File: tasks/create-pyxis-image/README.md
Comment (johnbieren) [inline tasks/create-pyxis-image/README.md:1]: Needs changes on line 20 and 21

**Review by johnbieren — COMMENTED** [review]
Just a few left

**Review by johnbieren — DISMISSED** [review]

**Comment by theflockers** [comment]
@jinqi7 can you help me to find out why is e2e failing for this PR?

**Comment by jinqi7** [comment]
> @jinqi7 can you help me to find out why is e2e failing for this PR?

There is no issue in e2e checking from my test.  https://redhat-internal.slack.com/archives/C04J47GL936/p1714461338691199?thread_ts=1714460928.326559&cid=C04J47GL936

**Review by johnbieren — APPROVED** [review]

**Pull request merged**